libraries/chain/controller.cpp

```diff
@@ -619,6 +619,9 @@ struct controller_impl {
    void init(std::function<bool()> check_shutdown, bool clean_startup) {
       uint32_t lib_num = (blog.head() ? blog.head()->block_num() : fork_db.root()->block_num);
 
+      EOS_ASSERT( lib_num >= this->conf.min_initial_block_num, misc_exception, "Controller head "
+      "at block number ${block_num}, which is smaller than the minimum required ${required}", ("block_num", fork_db.head()->block_num)("required",this->conf.min_initial_block_num) );
```

Comment:
This reports and uses the term head when lib is used. Also the variable name and option description should be clear that this feature is for lib and not head.